modules/olm-operatorhub.adoc

```diff
@@ -26,3 +26,36 @@ Community Operators::
 Optionally-visible software maintained by relevant representatives in the
 link:https://github.com/operator-framework/community-operators[operator-framework/community-operators]
 GitHub repository. No official support.
+
+OperatorHub itself is installed and run as an Operator by default on {product-title}.
+In particular, the OperatorHub runs in the
+openshift-marketplace namespace in OpenShift.
+
+Each operator in the OperatorHub is defined by two Custom Resource Definitions (CRDs):
```

Comment:
This is not correct .The CRDs are the ones marketplace watches for. Please see https://github.com/operator-framework/operator-marketplace#description